cli/cli/commands/import/import.go

```diff
@@ -0,0 +1,144 @@
+	}
+	file, err := os.Open(path)
+	if err != nil {
+		return stacktrace.Propagate(err, "File on '%v' was not found", path)
+	}
+	defer file.Close()
+
+	// Read the content of the file into a []byte slice
+	content, err := io.ReadAll(file)
+	if err != nil {
+		return stacktrace.Propagate(err, "Error reading file: %s\n", err)
+	}
+
+	script, err := convertComposeFileToStarlark(content)
+	if err != nil {
+		return stacktrace.Propagate(err, "Failed to convert compose to starlark")
+	}
+	logrus.Debugf("Generated starlark:\n%s", script)
+	err = runStarlark(ctx, script)
+	if err != nil {
+		return stacktrace.Propagate(err, "Failed to run generated starlark from compose")
+	}
+	return nil
+}
+
+func convertComposeFileToStarlark(content []byte) (string, error) {
+	// Load the Compose configuration from the []byte slice
+	config, err := loader.ParseYAML(content)
+	if err != nil {
+		return "", stacktrace.Propagate(err, "Error parsing YAML: %s\n", err)
+	}
+
+	// Convert the generic map to a structured compose.Config
+	project, err := loader.Load(types.ConfigDetails{ //nolint:exhaustruct
+		ConfigFiles: []types.ConfigFile{{Config: config}},
+	})
+	if err != nil {
+		return "", stacktrace.Propagate(err, "Error parsing docker compose")
+	}
+	script, err := convertComposeProjectToStarlark(project)
+	if err != nil {
+		return "", stacktrace.Propagate(err, "Error translating docker compose to Starlark")
+	}
+	return script, nil
+}
+
+// TODO(victor.colombo): Have a better UX letting people know ports have been remapped
+func convertComposeProjectToStarlark(compose *types.Project) (string, error) {
+	serviceStarlarks := map[string]string{}
+	for _, serviceConfig := range compose.Services {
+		portPiecesStr := []string{}
+		for _, port := range serviceConfig.Ports {
+			portStr := fmt.Sprintf("docker-%s=%d", port.Published, port.Target)
+			if port.Protocol != "" {
+				portStr += fmt.Sprintf("/%s", port.Protocol)
+			}
+			portPiecesStr = append(portPiecesStr, portStr)
+		}
+		starlarkConfig, err := add.GetServiceConfigStarlark(serviceConfig.Image, strings.Join(portPiecesStr, ","), serviceConfig.Command, serviceConfig.Entrypoint, nonSupportedField, nonSupportedField, emptyPrivateIpPlaceholder)
+		if err != nil {
+			return "", stacktrace.Propagate(err, "Error getting service config starlark for '%v'", serviceConfig)
+		}
+		serviceStarlarks[serviceConfig.Name] = starlarkConfig
+	}
+	script := "def run(plan):\n"
+	for serviceName, serviceConfig := range serviceStarlarks {
+		script += fmt.Sprintf("\tplan.add_service(name = '%s', config = %s)\n", serviceName, serviceConfig)
+	}
+	return script, nil
+}
+
+func runStarlark(ctx context.Context, starlarkScript string) error {
+	kurtosisCtx, err := kurtosis_context.NewKurtosisContextFromLocalEngine()
+	if err != nil {
+		return stacktrace.Propagate(err, "An error occurred connecting to the local Kurtosis engine")
+	}
+
+	enclaveCtx, err := kurtosisCtx.CreateEnclave(ctx, "docker-compose-import", false)
```

Comment:
is this a prefix? If you look at `run` you can see how a new one is generated

Reply:
Added a flag to autogenerate/set enclave name